configure.ac

```diff
@@ -136,7 +136,7 @@ dnl AC_FUNC_REALLOC
 AC_TYPE_SIGNAL
 AC_FUNC_STAT
 AC_FUNC_STRFTIME
-AC_CHECK_FUNCS([issetugid inet_pton])
+AC_CHECK_FUNCS([issetugid])
```

Comment:
FYI: line note comments might get lost after you push a commit which replaces the commit with the line notes.  Also, you can send off-topic, or tangentially-related questions, such as this, to me via email (see https://github.com/gstrauss/).

To answer your question: sorry, as is demonstrated in lighttpd-1.4.40-gcc2-std-gnu9x.patchset, lighttpd is compiled towards -std=gnu99.  I am considering changing that to -std=gnu11.  C99 was published over 17 years ago.  C11 was published almost 5 years ago.  Let that sink in for a moment. :)  I took a quick look at your patches and had all but forgotten about "not a static initializer" errors.  Also, I don't recall ever using a preprocessor that was unable to support token pasting (##).  While I am sure you have a reason for using gcc2, have you considered porting whatever needs to be ported into clang, if licensing with gcc is a sticking point?  We probably should continue this off-topic conversation offline.